src/OrleansTestingHost/TestClusterOptions.cs

```diff
@@ -22,8 +22,6 @@ public class TestClusterOptions
         /// <summary>Extended options to be used as fallbacks in the case that explicit options are not provided by the user.</summary>
         public class FallbackOptions
         {
-            /// <summary>Gets or sets whether the cluster will output traces in the console by default</summary>
-            public bool? TraceToConsole { get; set; }
```

Comment:
hmmm, I'm not so sure about getting rid of this, especially since the default was true... It's not completely straightforward to support though, but something we'll have to start doing for other properties too as we move to mainstream usage of SiloBuilder in TestCluster.
If not addressing yet, please add an issue to bring it back

Reply:
will add an issue

Reply:
but after this pr. traceToConsole will still be on by default. Since TestCluster is configured with a ConsoleLoggerProvider. But anyways I still created an issue #3429 for tracking,

Reply:
And there is no way to turn it off? :(
We had it off in Orleans' tests... all the more reason to bring this feature back

Reply:
well there's. just configure your test without the console logger provider ;) I think we should discuss this. Since I don't know how much orleans wants to deal with trace config from this point further on. We can put it back as a obsolete feature for sure.